.github/workflows/publish-helmcharts-to-artifactory.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: "Push Helm Charts to Artifactory"
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [master, v1.*]
+jobs:
+  BuildPublishHelmCharts:
+    env:
+      HELM_CHART_ARTIFACTORY_URL: "https://artifactory.magmacore.org:443/artifactory/"
+      HELM_CHART_MUSEUM_REPO: "helm-test"
+      HELM_CHART_MUSEUM_USERNAME: "${{ secrets.HELM_CHART_MUSEUM_USERNAME }}"
+      HELM_CHART_MUSEUM_TOKEN: "${{ secrets.HELM_CHART_MUSEUM_TOKEN }}"
+      MAGMA_ROOT: "${{github.workspace}}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      # Version is github job run number when running on master
+      # Or is branch name when on release branch
+      - name: Set helm chart version
+        run: |
+          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+            echo "VERSION=${GITHUB_RUN_NUMBER}" >> $GITHUB_ENV
+          else
+            echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          fi
+      - name: Launch build and publish script
+        run: |
+          orc8r/tools/helm/package.sh -d all -v $VERSION
+      # Notify ci-info channel when failing
```

Comment:
I think this deserves to be sent to the main CI channel -- it's a proper build failure, which we want to know about